downstream/titles/hub/configuring-private-hub-rh-certified/docinfo.xml

```diff
@@ -1,7 +1,7 @@
-<title>Managing Red Hat Ansible Content Collections and Ansible Galaxy collections in automation hub</title>
+<title>Managing Red Hat Certified and Ansible Galaxy collections in automation hub</title>
```

Comment:
It's probably best to change this in a separate PR - maybe even under a separate but linked ticket.
Felicia needs to be involved here because we need changes in Pantheon and she needs to file a ticket to get the "old" doc removed from the portal.

Reply:
Also, the titles in master.adoc and docinfo.xml must match